checker/src/main/java/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -648,4 +652,38 @@ public Void visitAnnotation(AnnotationTree node, Void p) {
         // All annotation arguments are non-null and initialized, so no need to check them.
         return null;
     }
+
+    @Override
+    protected TypeValidator createTypeValidator() {
+        return new NullnessValidator(checker, this, atypeFactory);
+    }
+
+    /**
+     * Nullness Validator. Check that primitive types are annotated with {@code @NonNull} even if
+     * they are the type of a local variable.
+     */
+    private static class NullnessValidator extends BaseTypeValidator {
+
+        /**
+         * Create Nullness validator.
```

Comment:
Nit. Make the `validator` capitalization consistent with the use in the class javadoc.

Reply:
I think it's OK to write "NullnessValidator" or "Nullness validator".  I wouldn't write "Nullness Validator", though "Nullness BaseTypeValidator" would also be OK.